tests/generators/run_generators_in_browser.js

```diff
@@ -51,8 +51,10 @@ async function runGeneratorsInBrowser() {
       args: ['--headless', '--no-sandbox', '--disable-dev-shm-usage', '--allow-file-access-from-files']
     };
   } else {
+    // --disable-gpu is needed to prevent Chrome from hanging on Linux with
+    // NVIDIA drivers older than v295.20.    
```

Comment:
Could you add a link here and below to #5345 so we have the future context, please?

Reply:
Done!